core/src/main/java/hudson/FilePath.java

```diff
@@ -2266,12 +2267,15 @@ private Integer writeToTar(File baseDir, DirScanner scanner, OutputStream out) t
 
     /**
      * Reads from a tar stream and stores obtained files to the base dir.
+     * @since TODO: the method uses 
```

Comment:
In such case <code>@since</code> allows to track major changes, which may be important to core developers. BTW, no problem if we remove this annotation